oci/python/image.yaml

```diff
@@ -0,0 +1,6 @@
+version: 1
+
+upload:
+  - source: canonical/chiselled-python
+    commit: a9fb59b6066cbe479ee2c395e316094b96af22b3
+    directory: python3.8/
```

Comment:
why isn't it being published?

Reply:
I am sorry, what do you mean by not published?

Reply:
You're missing the `release` field. Without it, this rock will be built but only kept in GHCR:

https://github.com/canonical/oci-factory/blob/796c8d44e61944de00b08f0de4701192d0ab8e8d/README.md?plain=1#L154-L157

Reply:
Updated in https://github.com/canonical/oci-factory/pull/125/commits/14fcd910957dc1ecf911b8472fea24fb52b8980f. Please have a look.